index.html

```diff
@@ -104,6 +106,9 @@
           <input type="text" placeholder="Exposure amount" onfocus="this.placeholder = ''" onblur="this.placeholder = 'Exposure amount' "id="input2"> Exposure.</p>
```

Comment:
The text above here probably needs to be changed, as it's now for the "seller" or the person wanting to be paid.